openllm-python/src/openllm/_service.py

```diff
@@ -6,6 +6,7 @@
 import _service_vars as svars
 import orjson
 
+from fastapi import FastAPI
```

Comment:
lets not do this. We don't depend on FastAPI. YOu can just use Starlette app for this